fitbenchmarking/fitbenchmark_one_problem.py

```diff
@@ -22,6 +22,10 @@
     from fitbenchmarking.fitting.controllers.scipy_controller import ScipyController
 except ImportError:
     ScipyController = None
+try:
```

Comment:
Imports should be alphabetical. It might also be worth keeping the controllers dict below in alphabetical order.

Reply:
I've changed this -- I've also added a line in the docs asking for them to be added alphabetically